services/github/src/utils.js

```diff
@@ -95,7 +95,7 @@ const shouldSkipPullRequest = ({ pull_request }) => {
  */
 const ansi2txt = (src) => {
   // eslint-disable-next-line no-control-regex
-  const regex = /\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]/gm;
+  const regex = /\x1B(\[[0-9;]*[JKGmsu]|\(B)/gm;
```

Comment:
Mostly correct, but I don't get where you see "literal `\` followed by `B`"? 
it is only `(B` sequence that needs to be matched 
Maybe it's also worth using some well-known library instead of doing it like this

Reply:
you're right, I mistyped!